core/domain/email_services.py

```diff
@@ -69,3 +75,165 @@ def get_feedback_thread_reply_info_by_user_and_thread_ids(user_id, thread_id):
     if model is None:
         return None
     return get_feedback_thread_reply_info_from_model(model)
+
+
+def get_incoming_email_address(reply_to_id):
+    """Gets the incoming email address. The client is responsible for recording
+    any audit logs.
+
+    Args:
+        reply_to_id: str. The unique id of the sender.
+
+    Returns:
+        str. The email address of the sender.
+    """
+    return 'reply+%s@%s' % (reply_to_id, feconf.INCOMING_EMAILS_DOMAIN_NAME)
+
+
+def is_email_valid(email_address):
+    """Determines whether an email address is invalid.
+
+    Args:
+        email_address: str. Email address to check.
+
+    Returns:
+        bool. Whether specified email address is valid.
```

Comment:
Write in complete English clauses. In this case, there's a missing "the" before "specified".

Check your other docstrings, too.